tests/libfixmath_unittests/Makefile

```diff
@@ -19,4 +17,5 @@ USEMODULE += printf_float
 
 TEST_ON_CI_WHITELIST += native
 
+include ../Makefile.tests_common
```

Comment:
Also using `BOARDS` before the include here.

The LINKFLAGS could be defined by a `LINKFLAGS += $(if $(filter native,$(BOARD)),-lm)`.